api/children/childrenModel.js

```diff
@@ -4,46 +4,31 @@ const getChildren = async () => {
   return await db('children');
 };
 
-const findByChildId = async (id) => {
+const findByChildId = async (child_id) => {
   return db('children')
     .join('profiles', 'children.profile_id', 'profiles.profile_id')
-    .where('child_id', id)
+    .where('child_id', child_id)
     .first();
 };
 
 const addChild = async (child) => {
   return await db('children').insert(child).returning('*');
 };
 
-const getEnrolledCourses = async (id) => {
-  // receives error if the id of the course doesn't exist exist  this have to modified later on we need to make a middleware that checks fot the existing courses then runs this model'
+const getEnrolledCourses = async (child_id) => {
```

Comment:
thank you for updating this function, now the child enrollments endpoint gives back much more useful data!